src/mappings/Tranche.ts

```diff
@@ -141,7 +153,7 @@ export function handleRedeemOrder(call: RedeemOrderCall): void {
   investorTx.currencyAmount = tokenPrice.gt(BigInt.fromI32(0))
     ? tb.pendingRedeemToken.times(tokenPrice).div(fixed27)
     : tb.pendingRedeemToken
-  investorTx.gasUsed = call.transaction.gasUsed
+  investorTx.gasUsed = call.transaction.gasLimit
```

Comment:
This is surprising. Why are we using `gasLimit` rather than ` gasUsed` here?

Reply:
This is a strange update they made in version 0.0.5 of the Mapping API:
<img width="868" alt="image" src="https://user-images.githubusercontent.com/6720265/175316936-faff5060-d33c-4c72-bdb1-3b7a88cacc6d.png">